packages/patternfly-4/react-table/src/components/Table/utils/decorators/collapsible.js

```diff
@@ -46,7 +51,7 @@ export const expandedRow = colSpan => {
   ) =>
     rowData.hasOwnProperty('parent') && {
       colSpan,
-      children: <ExpandableRowContent id={contentId + rowIndex}>{value.title || value}</ExpandableRowContent>
```

Comment:
This is not good change :disappointed: We can't change curren API of table and forcing consumers to update it will be bad practice. Do not add edit to children as of yet, just render them and disable editing of them. We can update the whole child render in follwup PR to make it easier.

Reply:
This change is easily fixable by just adding this line to first column

```js
    cellFormatters: [expandable] 
```

This shouldn't be such a big problem for users to adapt to it.

We really need this functionality in cnv and there is no possibility to render custom components in expandable row now.

Reply:
> We can update the whole child render in follwup PR to make it easier.

Can we discuss it in this PR instead?  this functionality is an important part of  `CollapsibleEditableTable` example.

Reply:
You are right, it makes it easier to have the changes separately. Let's resolve https://github.com/patternfly/patternfly-react/pull/1520 first